src/styles/pipelines.js

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+import { Button } from 'antd';
+
+import Colors from 'styles/colors';
+
+export const StyledPipelines = styled.div`
+  margin-top: 30px;
+`;
+
+export const StyledPipelineItem = styled.div`
+  position: relative;
+  width: 100%;
+  height: 72px;
+  background-color: ${Colors.White};
+  margin-bottom: 15px;
+`;
+
+export const StyledPipelineName = styled.div`
+  display: table-cell;
+  vertical-align: middle;
+  height: 72px;
+  width: 450px;
```

Comment:
are these necessary? 
```suggestion
  height: 72px;
  width: 450px;
```